jest.config.js

```diff
@@ -16,15 +16,19 @@ module.exports = {
       '<rootDir>/tests/fixtures/integration-docs/_platforms.json',
   },
   modulePaths: ['<rootDir>/src/sentry/static/sentry'],
-  modulePathIgnorePatterns: ['<rootDir>/src/sentry/static/sentry/dist'],
+
+  modulePathIgnorePatterns: [
+    '<rootDir>/src/sentry/static/sentry/dist',
+    '<rootDir>/tests/js/spec/views/userFeedback/index.spec.tsx',
```

Comment:
not sure the reason this test was in typescript. I added it here because it wasn't even being checked before (testMatch hadn't tsx) and it throws a lot of errors.